GAMMA/upgrade-assistant.clef

```diff
@@ -0,0 +1,346 @@
+{"@t":"2022-01-09T23:00:20.6374067Z","@mt":"Hosting starting","@l":"Debug","EventId":{"Id":1,"Name":"Starting"},"SourceContext":"Microsoft.Extensions.Hosting.Internal.Host"}
```

Comment:
I don't think this file is required - it's just a side effect of the upgrade, and I recommend we delete after review.